pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake, gnumake, glibc
+, enableJpeg ? false, libjpeg
```

Comment:
Any reason to add the flags instead of simply passing `libjpeg = null;` to `callPackage` (directly or via override)?